science/netcdf-cxx/Portfile

```diff
@@ -24,9 +23,9 @@ master_sites                ftp://ftp.unidata.ucar.edu/pub/netcdf/ \
                             http://www.gfd-dennou.org/arch/netcdf/unidata-mirror/ \
                             ftp://www.gfd-dennou.org/arch/netcdf/unidata-mirror/
 
-checksums           md5     d32b20c00f144ae6565d9e98d9f6204c \
-                    sha1    bab9b2d873acdddbdbf07ab35481cd0267a3363b \
-                    rmd160  69e9348ced7da8b6ddf08dad4c3a92c4f18d0648
+checksums           rmd160  69e9348ced7da8b6ddf08dad4c3a92c4f18d0648 \
+                    sha256  95ed6ab49a0ee001255eac4e44aacb5ca4ea96ba850c08337a3e4c9a0872ccd1 \
+                    size    703482
```

Comment:
why an update here of the `checksums`? It appears there was no `version` update so this seems incorrect or at least unexpected... or?

Reply:
@reneeotten For port **netcdf-cxx**, there are two other deleted checksum lines not shown here.  This looks like simply updating an obsolete checksum set with the current Macports set.  **md5/sha1/rmd160** is replaced by **rmd160/sha256/size**.  Yes there was no change in the version number, only the rev bump.  Agreed?

Reply:
Yes, indeed, @Dave-Allured and @reneeotten, the update of `netcdf-cxx` was only to remove the obsolete checksums and introduce the current ones. In fact, CI complained that sha256 was not included as checksum.